docs/data-discovery/tracking-catalog/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Data Discovery through the Tracking Catalog"
+date: "2022-10-26"
+sidebar_position: 0
+---
+
+
+## Introducing the Tracking Catalog
+
+
+Discovery and understanding of the data your organisation is tracking through Snowplow is essential to enable teams to unlock real value from that data.
+We know that in many cases, Data Analysts, Data Scientists and Product Managers struggle to find out what is being tracked, and what the data means. They normally have to rely on tribal knowledge and source this information directly with the front end developers who instrumented the tracking, or raise a request to the Data Team.
+
+However, through the Tracking Catalog, all users can have a holistic view of all their tracking in a always up-to-date and easy to consume view.
+
+
+* * *
+
+
+## Browsing the Tracking Catalog
```

Comment:
Can we make this more goal-oriented? Like “Understanding what data is tracked”? That would help a lot with search and SEO

Reply:
done